src/server/bootstrap/helmet.ts

```diff
@@ -0,0 +1,19 @@
+import helmet from 'helmet'
+import config from '../config'
+
+const cspOnlyReportViolations = config.get('cspOnlyReportViolations')
+const cspReportUri = config.get('cspReportUri')
+
+export default helmet({
+  contentSecurityPolicy: {
+    directives: {
+      defaultSrc: ["'self'"],
+      styleSrc: ["'self'", "'unsafe-inline'", 'https://fonts.googleapis.com/'],
+      fontSrc: ["'self'", 'https://fonts.gstatic.com/'],
+      imgSrc: ["'self'", 'data:'],
```

Comment:
`'data:'` from Go. Don't think we have inline base64 images

Reply:
Indeed we don't, so you can remove this.